openmicroscopy/bioformats
# Fake reader: add polygons polylines

See https://trello.com/c/SO7YDDHc/112-fakereader-implement-polygons-polylines

This PR adds a simple implementation for the `polygons` and `polylines` keys in the FakeReader.

To test this PR:
- check the Formats BSD unit tests are green with this PR included
- check the documentation page for generating test images lists polygons and polylines
- in the regions OMERO server, check the daily imported fake files with `polygons` and `polylines` render as expected using the client
